tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -332,12 +339,14 @@ Status FilterPipeline::filter_chunks_reverse(
   }
 
   if (total_size != tile.size()) {
-    return LOG_STATUS(
-        Status_FilterError("Error incorrect unfiltered tile size allocated."));
+    auto e = FilterPipelineStatusException(
+        "Error incorrect unfiltered tile size allocated.");
+    LOG_STATUS(e);
```

Comment:
I think we can skip LOG_STATUS here and just "throw FilterPipelineStatusException(..."

Reply:
Done